index.html

```diff
@@ -194,5 +194,31 @@ <h1>About me</h1>
       </div>
     </div>
   </section>
+
+  <footer class="footer">
+    <p>I'm always interested in hearing about new projects, 
+      so if you'd like to chat please get in touch.
+    </p>
+
+    <form action="https://formspree.io/f/xjvleokg" method="POST">
+      <input type="text" placeholder="Full Name" maxlength="30" required>
+      <input type="email" placeholder="Email Address" required>
+      <textarea name="textarea" placeholder="Enter text here" maxlength="500" required></textarea>
+      <button type="submit">Get in touch</button>
```

Comment:
- [ ] Please stick with Figma design the textarea shouldn't have a border-bottom, I know this is might sound silly but in the feature, you will work in a company and you must 100% stick with the designs they give to you 
your design : 
![image](https://user-images.githubusercontent.com/74744787/160809063-6d51942c-4ace-49ee-aa14-de5304219447.png)
Figma design : 
![image](https://user-images.githubusercontent.com/74744787/160809293-5297b816-0f2d-47d0-85d2-98e7d7d8c6f4.png)